spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/bedrock/cohere/BedrockCohereEmbeddingAutoConfiguration.java

```diff
@@ -37,12 +37,14 @@
  *
  * @author Christian Tzolov
  * @author Wei Jiang
+ * @author Soby Chacko
  * @since 0.8.0
  */
 @AutoConfiguration
 @ConditionalOnClass(CohereEmbeddingBedrockApi.class)
 @EnableConfigurationProperties({ BedrockCohereEmbeddingProperties.class, BedrockAwsConnectionProperties.class })
-@ConditionalOnProperty(prefix = BedrockCohereEmbeddingProperties.CONFIG_PREFIX, name = "enabled", havingValue = "true")
+@ConditionalOnProperty(prefix = BedrockCohereEmbeddingProperties.CONFIG_PREFIX, name = "enabled", havingValue = "true",
+		matchIfMissing = true)
```

Comment:
Because, currently,  we are using a single boot starter for all Bedrock providers, the Bedrock models should be [disabled by default](https://docs.spring.io/spring-ai/reference/api/bedrock-chat.html#_enable_selected_bedrock_model).